_episodes/08-bonus-episode.md

```diff
@@ -59,26 +59,25 @@ From: ubuntu:20.04
 As we see, several packages are installed and this installation is done over several layers.
 
 
-> ### Bonus question: Why is installation over several layers advantageous?
-{: .challenge}
-
-> ### Solution:
+> ## Bonus question: Why is installation over several layers advantageous?
+> ## Solution:
```

Comment:
Hmm, are you sure you don't need double `>` here? As in 
`> > ## Solution` etc.?

Reply:
You can probably find a similar example somewhere in another lesson

Reply:
> I think the bonus question layout is probably not yet correct... Does that work locally?

It worked locally. I was using the Bonus question template here: https://hsf-training.github.io/hsf-training-matplotlib/05-mplhep/index.html 
But it seems that we need to update that template as well. (I will open a new PR in matplotlib lesson.)

The current commit should work.

Reply:
Thanks!